bloggies_backend/src/data.sql

```diff
@@ -1,20 +1,19 @@
-DROP DATABASE IF EXISTS "bloggies";
+DROP DATABASE IF EXISTS "learning_circle_test";
```

Comment:
Do we maybe want just "learning_circle" here, so that in a dev environment we are directed to the right database? (see db.ts for logic)

Reply:
Yea, good point! I will change it to `learning_circle`. I'll also make some database commands to create the database used for the tests as well separately.

Reply:
Saw this! Ignore my other comment, thanks!